docker-compose.yml

```diff
@@ -57,7 +57,7 @@ services:
       -c pg_stat_statements.track=all
       -c pg_stat_statements.save=off
     ports:
-      - 127.0.0.1:15432:5432
+      - 127.0.0.1:5432:5432
```

Comment:
it was done because it was conflicting on my local system

Reply:
with local installed postgresql

Reply:
But do you use it?

My motivation was to make it behave the same as MySQL and MongoDB for local checks development:
```
pmm-admin add mysql
pmm-admin add mongodb
pmm-admin add postgresql postgresql 127.0.0.1:15432
```

Reply:
@BupycHuk ^